bitnami/metallb/templates/rbac.yaml

```diff
@@ -1,79 +1,16 @@
-{{- if .Values.rbac.create -}}
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: {{ include "metallb.fullname" . }}:controller
-  labels: {{- include "metallb.labels" . | nindent 4 }}
-rules:
-  - apiGroups:
-      - ''
-    resources:
-      - services
-    verbs:
-      - get
-      - list
-      - watch
-      - update
-  - apiGroups:
-      - ''
-    resources:
-      - services/status
-    verbs:
-      - update
-  - apiGroups:
-      - ''
-    resources:
-      - events
-    verbs:
-      - create
-      - patch
-  - apiGroups:
-      - policy
-    resourceNames:
-      - {{ include "metallb.fullname" . }}-controller
-    resources:
-      - podsecuritypolicies
-    verbs:
-      - use
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: {{ include "metallb.fullname" . }}:speaker
-  labels: {{- include "metallb.labels" . | nindent 4 }}
-rules:
-  - apiGroups:
-      - ''
-    resources:
-      - services
-      - endpoints
-      - nodes
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - ''
-    resources:
-      - events
-    verbs:
-      - create
-      - patch
-  - apiGroups:
-      - policy
-    resourceNames:
-      - {{ include "metallb.fullname" . }}-speaker
-    resources:
-      - podsecuritypolicies
-    verbs:
-      - use
+{{ if .Values.rbac.create }}
```

Comment:
Is this file still needed? I can see you are splitting the controller and speaker into different folders with their own rbac files.

Reply:
yep it is.
Because it is the same for both.
it's the role they share to look at the configmap.